### PR TITLE
Feature/40 educational materials

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [https://jira.epa.gov/browse/HMW-40](https://jira.epa.gov/browse/HMW-40)

## Main Changes:
* Added a new page that features educational materials.
* Included a link to the page in the banner icons (the graduation cap).
* Moved content management to the `LookupFilesContext` so it can be served from the S3 Bucket.

## Steps To Test:
1. Click the "Educators" navigation link
2. Check that the links are visible, in this case the "middleschool lesson plan" link

### CODE REVIEW

**Review by cschwinderg — APPROVED** [review]
Looks good! 

I'm going to hold off on merging this until all of the other PRs get merged. Maybe by the time that happens we will have some more content to add.

File: app/client/src/components/shared/Page.js
Comment (courtneymyers) [inline app/client/src/components/shared/Page.js:403]:
```suggestion
            title="Educators"
```

File: app/client/src/components/shared/AboutContent.js
Comment (courtneymyers) [inline app/client/src/components/shared/AboutContent.js:422]: No need to duplicate content – especially since this link is set in the `EducatorsContent` from the configurable JSON file. I'll replace all this duplicated education content defined here with what's defined in the `EducatorsContent` component (where you can, embrace composition over duplication).

**Review by courtneymyers — COMMENTED** [review]
Thanks, @maxdiebold-erg! I've left some comments below – I'm going to push up some changes based on those comments, but let's catch up tomorrow to discuss.

**Pull request merged**
